deploy-dev.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres:15.3-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgresPaSSword
+      POSTGRES_DB: cdb-dev
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+
+  backend:
+    build:
+      context: Backend
+    depends_on:
+      - postgres
+    restart: always
+    ports:
+      - 8080:8080
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:postgresql://postgres:5432/cdb-dev
+      SPRING_DATASOURCE_USERNAME: postgres
```

Comment:
Ove stvari ovise od osobe do osobe, što ne. Možda bi bilo bolje koristiti ${ENVIRONMENT_VARIABLE} tip podatka u yamlu. ${} označava da pokušava izvući iz enviromenta tu varijablu. Može se postaviti i da ima default. Također uvođenje .env file bi bilo korisno, ali sada nemamo dovoljno varijabli da me za to bude briga.

Reply:
Prilicno sam siguran da ne nego sta postavis gore u postgres containeru da se to koristi. Tj vazno je samo da je isto kao i u postgres containeru te nema veze za konfiguraciju na tvom actual racunalu.

Reply:
Slijep sam. Mislio sam da se spaja na tvoju bazu, ali sada vidim da si napravio zaseban postres kontenjer. Ispričavam se na brzopletom gledanju koda.